server/database/dataGeneration.js

```diff
@@ -0,0 +1,109 @@
+const moment = require('moment');
+const faker = require('faker');
+const csvWriter = require('csv-write-stream');
+const fs = require('fs');
+const writer = csvWriter();
+
+let totalRows = 0;
+writer.pipe(fs.createWriteStream('reviews.csv'));
+
+const generateUser = () => {
```

Comment:
Reducing conditionals here may speed up the process by a small fraction e.g. just generating a fake username for each user but I don't see anything wrong with this if the script runs in < 1 hr